ImageGallery/src/org/sleuthkit/autopsy/imagegallery/actions/OpenAction.java

```diff
@@ -119,9 +134,13 @@ public void setEnabled(boolean value) {
     }
 
     @Override
-    @NbBundle.Messages({"OpenAction.dialogTitle=Image Gallery"})
+    @NbBundle.Messages({"OpenAction.dialogTitle=Image Gallery",
+        "OpenAction.multiUserDialog.Header=Multi-user Image Gallery",
+        "OpenAction.multiUserDialog.ContentText=The Image Gallery updates itself differently for multi-user cases than single user cases. Notably:\n\n"
+        + "If your computer is analyzing a data source, then you will get real-time Image Gallery updates as files are analyzed (hashed, EXIF, etc.). This is the same behavior as a single-user case.\n\n"
+        + "If another computer in your multi-user cluster is analyzing a data source, you will get updates about files on that data source only when you launch Image Gallery, which will cause the local database to be rebuilt based on results from other nodes.",
```

Comment:
Perhaps clarify that "...you will get *Image Gallery* updates about the files..."?